practice-app/app/spectator.py

```diff
@@ -1,12 +1,14 @@
 import requests
-from flask import Flask, Blueprint, jsonify, abort, request
-import urllib
-from datetime import datetime, timedelta
-from math import cos, asin, sqrt, pi
+from flask import Blueprint, jsonify, abort, request
+from datetime import datetime
+from .dbinit import db, User, Eventpost
+from sqlalchemy.orm import sessionmaker
+import time
+Session = sessionmaker(db)
+session = Session()
 
-spectator_api = Blueprint('spectator_api', __name__)
+player_api = Blueprint('player_api', __name__)
```

Comment:
This line should be the line below because it's part of the spectator API not player API.
```suggestion
spectator_api = Blueprint('spectator_api', __name__)
```